ci/matrix.yaml

```diff
@@ -51,7 +51,8 @@ workflows:
     - {jobs: ['test'],  ctk: '11.1', gpu: 'v100',     sm: 'gpu', cxx: 'gcc6',    std: [11]}
     - {jobs: ['test'],  ctk: '11.1', gpu: 't4',       sm: 'gpu', cxx: 'clang9',  std: [17]}
     - {jobs: ['test'],  ctk: '11.8', gpu: 'rtx2080',  sm: 'gpu', cxx: 'gcc11',   std: [17]}
-    - {jobs: ['test'],  ctk: 'curr', gpu: 'rtxa6000', sm: 'gpu', cxx: 'gcc7',    std: [14]}
+    - {jobs: ['test'],  ctk: 'curr', gpu: 'a100',     sm: 'gpu', cxx: 'gcc7',    std: [11], cpu: 'arm64'}
+    - {jobs: ['test'],  ctk: 'curr', gpu: 'rtxa6000', sm: 'gpu', cxx: 'gcc9',    std: [14]}
```

Comment:
Is there a reason we test lowest gcc and C++11?

Id rather have gcc11 and [all]

Reply:
The goal with these builds is to test on different hardware, not different dialects. I don't want to do `all` to keep the number of jobs under control, instead the stds are rotated across the cards.